### ISSUE TITLE
After the deletion timer has expired, the current view shifts.

### ISSUE THREAD

**Issue by Somebodyisnobody** [nextcloud/contacts#1375]
### Describe the bug
When I delete a contact I get a recovery prompt for a while. If I select another contact from the list below during this time and the timer expires, the browser switches to another contact because the list has updated. This is very annoying when editing and sorting multiple contacts in a row.

**To Reproduce**
1. Create a few contacts
2. Delete the first or second
3. Scroll down and select another contact close after step 2
4. wait a moment

**Expected behavior**
The Browser should keep the selected contact selected

**Actual behavior**
After the deletion timer has expired, the current view shifts and another contact is being selected.

**Screenshots**
Not applicable

### Server configuration

**Operating system**: Ubuntu

**Web server:** Nginx

**Database:** MariaDB

**PHP version:** 7.3

**Nextcloud version:** 17.0.1

**Contacts version:** 3.1.6

**Updated from an older Nextcloud or fresh install:** up

**Signing status:** OK

**List of activated apps:**
Not nearcessy

**Nextcloud configuration:**
Not nearcessy

### Client configuration
**Browser:** FF 71

**Operating system:** Windows

**CardDAV-clients:** -

#### *Web server error log*
Not nearcessy, UI problem

#### *data/nextcloud.log*
Not nearcessy, UI problem

#### Browser log
<details>
<summary>How to access your browser console (Click to expand)</summary>
Didn't get it to work

**Comment by jancborchardt** [comment]
This is a sort-of duplicate of "Contact deletion takes long / undo is too present" #1118

Also this would be fixed as a side-effect of removing the delete icon in the contact list: https://github.com/nextcloud/contacts/issues/1368#issuecomment-570118109 :)

**Comment by Somebodyisnobody** [comment]
@jancborchardt maybe remove the label needs triage just for the protocol
